wrfhydropy/core/ensemble.py

```diff
@@ -54,10 +54,114 @@ def parallel_run(arg_dict):
     else:
         os.chdir(str(pathlib.Path(arg_dict['ens_dir']) / arg_dict['member'].run_dir))
     mem_pkl = pickle.load(open("WrfHydroSim.pkl", "rb"))
+
     mem_pkl.run()
     return mem_pkl.jobs[0].exit_status
 
 
+def parallel_teams_run(arg_dict):
+    """Parallelizable function for teams to run an EnsembleSimuation."""
```

Comment:
You'll have to walk me through this, I'm kind of lost. All I really have to say about that then is we need to document it well.